lib/tasks/demo_data_for_development.rake

```diff
@@ -0,0 +1,70 @@
+namespace :data do
+  desc 'Create demo data for our local development'
+  task test: :environment do
+    include FactoryGirl::Syntax::Methods
+
+    def generate_events program, conference
```

Comment:
Maybe I don't know. But look at it like this we have covered all possible cases for our conferences and we can check all possible type of events in each of this conference.:wink: